api/db-cdr/generate-cdr/build-search-all-events.sh

```diff
@@ -388,7 +388,9 @@ WHERE o.observation_concept_id is not null
               where domain_id = 'Observation'
               and vocabulary_id in ('CPT4')
               and standard_concept = 'S'
-            )"
+            )
+    and c.vocabulary_id != 'PPI'
+    and c.concept_class_id != 'Survey'"
```

Comment:
Found that we were inserting standard rows for surveys. This is not needed, so added check to stop this.